calculated_policies/aws_vpc_public_routes/README.md

```diff
@@ -0,0 +1,122 @@
+# AWS EC2 - Restrict Instance images to trusted AWS accounts AMIs
```

Comment:
Documentation for this is out of sync, please review.

Reply:
fixed in newest commit